plugins/CoreHome/javascripts/uiControl.js

```diff
@@ -67,6 +75,9 @@
                     throw new Error("Error: " + control.constructor.name + "'s destroy method does not call " +
                                     "UIControl.destroy. You may have a memory leak.");
                 }
+            } else {
+                // Control is still active / used.
+                activeControls.push(control);
             }
         }
```

Comment:
Nope.

`var activeControls = UIControl._controls = [];` sets both `UIControl._controls` and `activeControls` to the same new array instance.

`activeControls.push` modifies that same array instance.